spring-core/src/test/java/facade/BookingFacadeImplTest.java

```diff
@@ -0,0 +1,154 @@
+package facade;
+
+import model.Event;
+import model.Ticket;
+import model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.List;
+import java.util.Objects;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(locations = "classpath:beans.xml")
+public class BookingFacadeImplTest {
+
+    @Autowired
+    private BookingFacade bookingFacade;
+
+    @Test
+    public void Should_Pass_When_UserIsRetrieved() {
```

Comment:
for all methods lets keep with name conventions like here https://dzone.com/articles/7-popular-unit-test-naming

Reply:
I used this pattern Should_ExpectedBehavior_When_StateUnderTest

Reply:
in this case you have not needed prefix should_pass, if you have test case it is obviously clear what `should pass`

better story this is type 1 for test case: MethodName_StateUnderTest_ExpectedBehavior - getUserById_obtainUserInfo

anyway your name is not clear for this

ShouldNotNullUser_whenGetUserByIdInvoked

or